Automattic/jetpack
# Debug Helper: set current primary user

#### Changes proposed in this Pull Request:
* Add a button to set the current user ID as connection's primary user

#### Other information:


#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Go to "Jetpack Debug -> Broken Token"
2. Click "Clear the Primary User"
3. Click "Set current user as Primary User"
4. Confirm that the "Primary User" value matches your user's ID.
